### PR TITLE
Avoid settimeout in favour of readFile

### PR DESCRIPTION
**Pull request by bas-ie**
See what you think of this as an alternative to our current content. Main objective is to keep focused on practical examples from the filesystem that they can then put to work in routes in the coming days.

### CODE REVIEW

**Comment by joshuavial** [comment]
Happy with this @richchurcher - we've been just using two arguments on fs.readFile so students get a buffer and have to think about the type instead of 'utf8' as the second param. No need to change, just an FYI

**Comment by richchurcher** [comment]
Yeah, I thought about it but on balance decided it was better not adding more complexity if it could be avoided!

**Comment by don-smith** [comment]
It's a shame to lose the pizza metaphor. Based on student responses, that seemed to really resonate with most of them. Overall, I really like these changes. Definitely an improvement. But can we keep a _real world metaphor_ section before the filesystem section?

**Comment by richchurcher** [comment]
On it!

**Comment by richchurcher** [comment]
How's this?

I note that there's some crossover with the filesystem concept. However the fs concept uses buffers, which is a useful contrast. On balance, I think I'll take writeFile back out of this one and save it for the next day.

File: async/callbacks.md [hunk 1]
```diff
@@ -22,89 +26,68 @@ function deliver(err, pizza) {
...
-  console.log('starting work')
-  setTimeout(finishWork, 100)
-  console.log('now working')
-}
+var fs = require('fs')
 
-function finishWork () {
-  console.log('finished work')
-}
+fs.readFile('animals.txt', 'utf8', function (err, data) {
```
Comment (don-smith) [inline async/callbacks.md]: Perhaps `fileContents` would be a better parameter name than `data`?

**Comment by richchurcher** [comment]
Removed Express stuff given the move to make Monday an async/fs day without Express.

**Pull request merged**